src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/methodparampad/InputMethodParamPadWhitespaceAround.java

```diff
@@ -11,11 +11,11 @@
 package com.puppycrawl.tools.checkstyle.checks.whitespace.methodparampad;
 
 @SuppressWarnings({"this", "that"})
-public class InputMethodParamPadWhitespaceAround // ok
+public class InputMethodParamPadWhitespaceAround
 {
     protected InputMethodParamPadWhitespaceAround ( int i )
     {
-        this (); //whitespace
+        this ();
```

Comment:
Change it to `//whitespace after 'this' so ok`

Reply:
This check ignores explicit constructor invocation: https://github.com/checkstyle/checkstyle/issues/13675

Reply:
Added what should be a violation to this input file with `until...` comment.